<launch>

    <!-- 启动相机节点  -->
    <!-- <include file="$(find astra_camera)/launch/astra.launch" /> -->
    <include file="$(find usb_cam)/launch/usb_cam-test.launch"/>

    <!-- 启动人眼识别节点  -->
    <node pkg="castlex_vision_detector" name="astra_cv_close_eye_detect" type="astra_cv_close_eye_detect.py" output="screen">
        <remap from="input_rgb_image" to="/usb_cam/image_raw" />      
        <rosparam>
            scaleFactor: 1.2
            minNeighbors: 2
            minSize: 40
            maxSize: 60
        </rosparam>
        <param name="cascade_1" value="$(find castlex_vision_detector)/data/haar_detectors/haarcascade_frontalface_alt_eye.xml" />
        <param name="cascade_2" value="$(find castlex_vision_detector)/data/haar_detectors/haarcascade_profileface.xml" />
        <param name="cascade_3" value="$(find castlex_vision_detector)/data/haar_detectors/haarcascade_eye_tree_eyeglasses.xml" />
    </node>

    <!-- 启动订阅cv_bridge_image话题  -->
    <node name="rqt_image_view" pkg="rqt_image_view" type="rqt_image_view" args="/cv_bridge_image"/>

</launch>
